Update Expense Tracker in UserGuide, Glossary and NFR in DeveloperGuide (#102)
AY1920S2-CS2103T-W17-3/main: docs/DeveloperGuide.adoc

[appendix]
== Non Functional Requirements

> * Application should work on any mainstream OS as long as it has Java 11 or above installed.

Reviewer: You can set a reference for mainstream OS to point to 'mainstream OS' in the glossary since inline anchors are already set up for all the terms.
```suggestion
* Application should work on any <<mainstream-os,mainstream OS>> as long as it has Java 11 or above installed.
```

* Application should be able to hold up to 20 trips no longer than 1 month each without a noticeable sluggishness in performance for typical usage.

* Application should respond within 2 seconds of the client's query.

* A user with above-average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.

* Application should be easy to use for a new user when following the User Guide.

* Application should work without requiring an installer.

* Application should not depend on a remote server.

* Application should be for a single user i.e. (not a multi-user product).

* Application should not require an online connection.

_{More to be added}_

[appendix]
== Glossary

[[e-t]] E.T.::
An abbreviation for Easy Travel, the name of the application.

[[mainstream-os]] Mainstream OS::
Windows, Linux, Unix, macOS.

[[prefix]] Prefix::

> The term that comes before each parameter in the command. For example, the prefix in p/Number is p/.

Reviewer: Can consider changing the prefix to one that we use in our application. e.g. `country/COUNTRY`
```suggestion
The term that comes before each parameter in the command. For example, the prefix in country/COUNTRY is country/.
```

[[prefix-name]] Prefix Name::

> The word that comes before ‘/’ in the prefix. For example, the prefix name of p/ is p.

Reviewer: Same as above, you can consider changing the prefix to one that we use in our application. e.g. country/COUNTRY
```suggestion
The word that comes before ‘/’ in the prefix. For example, the prefix name of country/ is country.
```
